ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
+    whisk_api_host={{ whisk_api_host_name | groups['edge'] | first }}
```

Comment:
I'm not familiar with `whisk_api_local_host_name` but it seems like perhaps we should use that for both installing the actions and for URL calculation. This would simplify things a bit, so long as it is always correct to use it for those purposes. @rabbah thoughts?